cmd/zt_copy_s2smigration_test.go

```diff
@@ -264,10 +264,20 @@ func (s *cmdIntegrationSuite) TestS2SCopyFromS3ToBlobWithBucketNameNeedBeResolve
 	rawDstBlobServiceURLWithSAS := scenarioHelper{}.getRawBlobServiceURLWithSAS(c)
 	raw := getDefaultRawCopyInput(rawSrcS3BucketURL.String(), rawDstBlobServiceURLWithSAS.String())
 
-	// bucket should be resolved, and objects should be scheduled for transfer
+	// bucket should not be resolved, and objects should not be scheduled for transfer
 	runCopyAndVerify(c, raw, func(err error) {
 		c.Assert(err, chk.NotNil)
```

Comment:
I'm puzzled why we get objects not scheduled for transfer here.  I understand why we get the glcm warning, and its nice that the test checks for it.  But if the bucket name can be resolved (by changing  the -- to -2- as noted in comment above) shouldn't all the transfers actually be scheduled?  I have a hunch I'm missing something... but can't think what!

Reply:
Only one bucket is targeted: One with an invalid name. The destination is a service, which means the container will be created before the transfer.

If container creation fails, no transfers have been scheduled.